web-ng/src/app/components/layer-dialog/layer-dialog.component.html

```diff
@@ -96,6 +96,11 @@
               Add option
             </a>
           </div>
+          <div fxLayout fxLayoutAlign="end" class="question-actions">
+            <span class="required-label"> Required </span>
```

Comment:
Can we use CSS here to add padding instead of " "?

Reply:
I am adding margin-right here, forgot to remove extra spaces around the "Required"